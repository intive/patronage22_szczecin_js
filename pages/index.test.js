/* globals describe, it, expect  */
/* eslint-env jest */

import { render, waitFor } from '@testing-library/react'
import Home from './index.page'

describe('Home', () => {
  const tilesList = [
    {
      id: 1,
      name: 'mockedName',
      date: '1 Jan 2022',
      cardCount: 0,
      hasPassword: false
    }
  ]

  it('should render Home page', async () => {
    const { container } = render(<Home tilesList={tilesList} />)
    await waitFor(() => { expect(container).toMatchSnapshot() })
  })
})
