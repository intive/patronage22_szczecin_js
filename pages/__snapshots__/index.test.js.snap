// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render Home page 1`] = `
<div>
  <div
    class="container"
  >
    <header
      class="sc-eCImPb jEdnXJ"
    >
      <div
        class="sc-jRQBWg dquuTs"
      >
        <a
          class="sc-gKclnd dkHLmp"
          href="/"
        >
          <span
            class="material-icons"
          >
            crop_original
          </span>
          <p
            class="sc-iCfMLu igRBkW"
          >
            headerTitle
          </p>
        </a>
      </div>
    </header>
    <main
      class="sc-hKwDye gmyMrp"
    >
      <div
        class="sc-dkPtRN hzCcxk"
        role="button"
        tabindex="0"
      >
        <button
          class="sc-bdvvtL jESiNr"
        >
          <span
            class="material-icons sc-gsDKAQ jUyrhl"
          >
            crop_original
          </span>
          New board
        </button>
      </div>
      <div
        class="sc-furwcr cGXmQV"
      >
        <div
          class="sc-pVTFL jeqYEW"
        >
          <span
            class="material-icons"
          >
            lock_outlined
          </span>
        </div>
        <h4
          class="sc-jrQzAO fKSzgk"
        >
          Board name 1
        </h4>
        <p
          class="sc-kDTinF sc-iqseJM etjriW cKNRkB"
        >
          1 Jan 2022
        </p>
        <p
          class="sc-kDTinF etjriW"
        >
          cardCount
        </p>
      </div>
      <div
        class="sc-jRQBWg ctTUaR"
      >
        <div
          class="sc-gKclnd bhZFuz"
        >
          <span
            class="material-icons"
          >
            lock_outlined
          </span>
        </div>
        <h4
          class="sc-iCfMLu iYsejJ"
        >
          Board name 1
        </h4>
        <p
          class="sc-furwcr sc-pVTFL jzezhb bOqEUQ"
        >
          1 Jan 2022
        </p>
        <p
          class="sc-furwcr jzezhb"
        >
          cardCount
        </p>
      </div>
      <div
        class="sc-jRQBWg ctTUaR"
      >
        <div
          class="sc-gKclnd bhZFuz"
        >
          <span
            class="material-icons"
          >
            lock_outlined
          </span>
        </div>
        <h4
          class="sc-iCfMLu iYsejJ"
        >
          Board name 1
        </h4>
        <p
          class="sc-furwcr sc-pVTFL jzezhb bOqEUQ"
        >
          1 Jan 2022
        </p>
        <p
          class="sc-furwcr jzezhb"
        >
          cardCount
        </p>
      </div>
    </main>
    <br />
    <button
      class="sc-bdvvtL jESiNr"
    >
      Continue
    </button>
    <br />
    <button
      class="sc-bdvvtL jESiNr"
      disabled=""
    >
      Continue
    </button>
    <br />
    <button
      class="sc-bdvvtL jESiNr"
    >
      <span
        class="material-icons sc-gsDKAQ jUyrhl"
      >
        crop_original
      </span>
      Continue
    </button>
    <br />
    <button
      class="sc-bdvvtL jESiNr"
      disabled=""
    >
      <span
        class="material-icons sc-gsDKAQ jUyrhl"
      >
        crop_original
      </span>
      Continue
    </button>
    <br />
    <button
      class="sc-bdvvtL dzUoKp"
    >
      Continue
    </button>
    <br />
    <button
      class="sc-bdvvtL dzUoKp"
      disabled=""
    >
      Continue
    </button>
    <br />
    <button
      class="sc-bdvvtL dzUoKp"
    >
      <span
        class="material-icons sc-gsDKAQ jUyrhl"
      >
        crop_original
      </span>
      Continue
    </button>
    <br />
    <button
      class="sc-bdvvtL dzUoKp"
      disabled=""
    >
      <span
        class="material-icons sc-gsDKAQ jUyrhl"
      >
        crop_original
      </span>
      Continue
    </button>
    <br />
  </div>
</div>
`;
